### DISCUSSION TITLE
Any way to migrate database from mysql to postgresql?

### DISCUSSION THREAD

**Discussion by mobeicanyue** [umami-software/umami#2533] [Q&A]
I use mysql as umami's database, but now I want to migrate it to serverless plateform anyway to migrate it? I think their syntax are similar.

**Comment by mobeicanyue** [comment]
I use pgloader to migrate, but encounter `mysql: 76 fell through ECASE expression.`

**Comment by mobeicanyue** [comment]
Using  pgloader

**Reply by kaido2ki** [reply]
how did you do this? can you share your pgloader config?

**Reply by mobeicanyue** [reply]
> how did you do this? can you share your pgloader config?

This plan isn't ideal. Better to create a new database; the old backups will do.
